### ISSUE TITLE
About the paras in blender rendering script

### ISSUE THREAD

**Issue by s1752729916** [Mrguanglei/Instantmesh_scriptData#1]
hi,
Thanks for sharing the rendering script for instantmesh. 
When I was creating my own dataset, I found that the dataset parameters of Instantmesh were inconsistent with OpenLRM, including camera FOV, camera distance, etc. Have you tested your rendering script to be consistent with Instantmesh? My testing method is to compare the 3D models generated using the ground-truth multi-view input from my own dataset and the zero123++input from InstantMesh. I found that the 3D models generated using OpenLRM parameters are worse than those generated from InstantMesh.

**Comment by Mrguanglei** [comment]
Yes, I made a change in openlrm blender_script, which is exactly the same as instantmesh

**Comment by s1752729916** [comment]
thanks for your reply.
can you tell me what parameters you have changed? I see that the camera distance, FOV, and scene scale are consistent with OpenLRM
